README.md

```diff
@@ -1,93 +1,58 @@
-serving as a single-node low-latency runtime with explicit networking. This allows us to support
-not just data processing pipelines, but distributed protocols (e.g. Paxos) and real-world
-long-running applications as well.
+Hydro is a novel distributed programming library for standard Rust. Hydro allows developers to build distributed systems that are efficient, scalable, and correct. 
 
-Take a look at the [Hydroflow Book](https://hydro.run/docs/hydroflow/).
+Hydro integrates naturally into standard Rust constructs and IDEs, providing types and programming constructs for ensuring distributed correctness. Under the covers it provides a metaprogrammed compiler that optimizes for cross-node issues of scaling and data movement while leveraging Rust and LLVM for per-node performance. 
 
-## The Hydroflow Surface Syntax
+We often describe Hydro via a metaphor:*LLVM for the cloud*. Like LLVM, Hydro is a staged compilation framework with a low-level Internal Representation language. In contrast to LLVM, Hydro focuses on distributed aspects of modern software. 
+  
+<div align="center">
+  <img src="docs/static/img/hydro-stack.png" alt="Image description" width="200">
+</div>
 
-Hydroflow comes with a custom "surface syntax"—a domain-specific language which serves as a very
-simple, readable IR for specifying single-node Hydroflow programs. These programs are intended to be stitched together
-by the Hydro stack to create larger autoscaling distributed systems.
 
-Here's a simple example of the surface syntax. Check out the [Hydroflow Playground](https://hydro.run/playground)
-for an interactive demo.
-```rust
-source_iter(0..10)
-  -> map(|n| n * n)
-  -> filter(|n| *n > 10)
-  -> foo;
+## The Language (and the Low-Level IR)
+Hydro provides a high-level language for the majority of developers called [Hydroflow+](https://hydro.run/docs/hydroflow_plus). Hydroflow+ allows you to program an entire fleet of processes from a single program, and then launch your fleet locally or in the cloud via [HydroDeploy](https://hydro.run/docs/deploy). Get started with Hydroflow+ via the Hydroflow+ [documentation](https://hydro.run/docs/hydroflow_plus) and [examples](https://github.com/hydro-project/hydroflow/tree/main/hydroflow_plus_test/examples).
 
-foo = map(|n| (n..=n+1))
-  -> flatten()
-  -> for_each(|n| println!("Howdy {}", n));
-```
+> Internally, the Hydro stack compiles Hydroflow+ programs into a low-level Internal Representation (IR) language called [Hydroflow](https://hydro.run/docs/hydroflow); each process corresponds to a separate Hydroflow program. In rare cases you may want to compose one or more processes in Hydroflow by hand; see the Hydroflow [documentation](https://hydro.run/docs/hydroflow) or [examples](https://github.com/hydro-project/hydroflow/tree/main/hydroflow/examples) for details.
```

Comment:
There is no API for such "rare cases", rather I would focus that the Hydroflow+ compiler emits snippets in this IR to abstract away from low-level concerns such as scheduling.

Reply:
There's an entire DSL for such "rare cases", we've got docs right here for it --- more extensive than the HF+ docs at the moment, so it would seem to merit mention -- and we have written some non-trivial things with it. For now, at least, I'd like to maintain HF as an artifact worthy of mention.